### DISCUSSION TITLE
Is there any way to pass variables as Json string not an object

### DISCUSSION THREAD

**Discussion by Vaulz** [graphql-dotnet/graphql-client#656] [Q&A]
I'm trying to call query, and I have my variable parameters as json string. I can't desirialize them to some type cause service should be abstract.
Something like this:
```
var request = new GraphQLHttpRequest
        {
            Query = Query,
            Variables = "{\"skip\":0,\"take\":100}",
            OperationName = OperationName
        };
```

**Comment by rose-a** [comment]
The problem is that the string is run through the JSON serializer again, which most likely escapes it again.

You could propably work with a `Dictionary<string,object>`, anonymous objects or build your JSON object on the fly using `JToken` or the respective objects in `System.Text.Json`...
